tests/functional/build/test_build.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+
+from dbt.tests.util import run_dbt, copy_file
+
+
+from tests.functional.build.fixtures import (  # noqa: F401
+    snapshots,
+    tests_failing,
+    models,
+    models_failing,
+    seeds,
+    models_circular_relationship,
+    models_simple_blocking,
+    test_files,
+    models_interdependent,
+    project_files,
+)
+
+
+# from test.integration.base import DBTIntegrationTest, use_profile, normalize
+from test.integration.base import normalize
+import yaml
+import os
+
+
+class TestBuildBase:
```

Comment:
Only the classes that contain tests should start with `Test`.  Maybe just name this `BuildBase`?

Reply:
Sure.  @ChenyuLInx this came from the conversion script so it's probably worth updating.

Reply:
We could try to find the class that doesn't have a test function and change its name but feels like this might be tricky and going over the tests before merging could catch it easily.

Reply:
@ChenyuLInx Yeah that makes sense, maybe just add it as a thing to check for in the docs?  /shrug